documentation/modules/configuring/con-configuring-topic-operator.adoc

```diff
@@ -7,6 +7,10 @@
 
 [role="_abstract"]
 Use `topicOperator` properties in `Kafka.spec.entityOperator` to configure the Topic Operator.
+
+NOTE: If you are using the preview of unidirectional topic management, the following properties are not used and will be ignored: `zookeeperSessionTimeoutSeconds` and `topicMetadataMaxAttempts`.
```

Comment:
Should we use full path here to make it more clear what does it relate to?

```suggestion
NOTE: If you are using the preview of unidirectional topic management, the following properties are not used and will be ignored: `Kafka.spec.entityOperator.topicOperator.zookeeperSessionTimeoutSeconds` and `Kafka.spec.entityOperator.topicOperator.topicMetadataMaxAttempts`.
```

Or maybe make it more general as it applies also when deploying the Topic Operator in the standalone mode.